scripts/superdesk-desks/desks.js

```diff
@@ -172,7 +172,7 @@
                 }
 
                 scope.$watch('filter', queryItems);
-                scope.$on('task:stage', function(event, data) {
+                scope.$on('task:stage', function($event, data) {
```

Comment:
ok great, I will change all the relevant unused argument name respecting that naming convention.